corehq/apps/geospatial/forms.py

```diff
@@ -75,6 +77,18 @@ class Meta:
         required=False,
         min_value=1,
     )
+    max_case_distance = forms.IntegerField(
+        label=_("Max distance (km) to case"),
+        help_text=_("The maximum distance (in kilometer) from the user to the case. Leave blank to skip."),
+        required=False,
+        min_value=1,
+    )
+    max_case_travel_time = forms.IntegerField(
+        label=_("Max travel time (minutes) to case"),
+        help_text=_("The maximum driving time (in minutes) from the user to the case. Leave blank to skip."),
+        required=False,
+        min_value=0,
```

Comment:
Having a value of 0 feels like the same thing as simply leaving it blank to skip. Would it make more sense to rather have this be a `min_val` of 1?

Reply:
The problem with having a min value of 1 is that there might be cases which is situated less than 1 min away, so I don't want to potentially exclude those.

Reply:
That is a good point, I didn't think of that scenario. This makes sense to leave as is then.